articles/atom-vs-visual-studio-code-functionality-and-features-comparison/index.md

```diff
@@ -0,0 +1,178 @@
+### Introduction
+
+This article compares the Atom and Visual Studio Code editors based on features and comparison. There are loads of code editors, but we will review Atom and Visual Studio Code editors in this article. These source code editors tend to increase the developer's productivity by providing different plugins and components. There are various features such as code-snippets, syntax highlighting, preview options, compiling, and debugging.
+
+Comparing these two text editors will enable developers to choose the editor they would love to use as per their requirement.
+
+### Table of contents
+
+- [Atom](#atom)
+
+- [Visual Studio Code Editor](#visual-studio-code-editor)
+
+- [Atom vs. VS Code: Tabular Comparison](#atom-vs-VS-code:-tabular-comparison)
+
+- [Atom vs. VS Code: Features and Functionality Comparison](#atom-vs-VS-code:-features-and-functionality-comparison)
+
+### Atom
+
+[Atom](https://atom.io/) is a free and open-source text editor developed by GitHub. It works across Windows, Linux, and Mac OS. Currently, the Atom is at version 1.52.0. It lets you easily customize every aspect of it to speed up the workflows. Atom uses a framework called Electron. Electron is a Node. js based framework that allows cross-platform in an application setting.
```

Comment:
> Electron is a Node. js based framework that allows cross-platform in an application setting.

Electron is a JavaScript framework that enables building cross-platform desktop applications.